Maple2.Server.Game/Packets/ItemBoxPacket.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Maple2.Model.Error;
+using Maple2.PacketLib.Tools;
+using Maple2.Server.Core.Constants;
+using Maple2.Server.Core.Packets;
+
+namespace Maple2.Server.Game.Packets;
+
+public static class ItemBoxPacket {
+
+    public static ByteWriter Open(int sourceItemId, int amount, ItemBoxError error) {
```

Comment:
Can we split this into 2 functions, when that is never error and one that is always error? Or maybe have a default value for error so you don't need to pass it in.

Reply:
As far as I know, we would always have to pass it in.